vendor/grpc-1_13_1/build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.beam.gradle.GrpcVendoring
+
+apply plugin: org.apache.beam.gradle.VendorJavaPlugin
+
+description = "Apache Beam :: Vendored Dependencies :: gRPC :: 1.13.1"
+
+group = "org.apache.beam"
+version = "0.1"
```

Comment:
Can you document how the versioning of this artifact should be maintained? I suspect the rules are something like:

- The version embedded in the vendored artifact ID should match exactly the version of vendored library (`1.13.1`). Upgrading the version of vendored library should trigger a change in the artifact name.
- Vendored artifact versioning starts at `0.1` and is decoupled from Beam releases and version numbers 
- Increment the vendored artifact version revision only if we need to release a new version.

Reply:
I documented this in the VendoringJavaPlugin and referenced that within these projects.